src/native/nb_native.gyp

```diff
@@ -16,15 +16,24 @@
             }],
             [ 'node_arch=="arm64"', {
                 'cflags': ['-DUSE_NEON']
-            }],
+            }]
         ],
     },
 
     'targets': [{
+	'variables': {
+            'BUILD_S3SELECT%':0
```

Comment:
What is the sign %?

Reply:
This is GYP syntax for default variable values.
https://gyp.gsrc.io/docs/InputFormatReference.md#Providing-Default-Values-for-Variables